src/Core/src/Platform/iOS/ContentView.cs

```diff
@@ -114,9 +136,28 @@ void SetClip()
 			else
 				path = clipShape?.PathForBounds(bounds);
 
-			var nativePath = path?.AsCGPath();
+			var platformClipPath = path?.AsCGPath();
+
+			return platformClipPath;
+		}
+
+		bool RequireClip(CGPath? platformClipPath)
```

Comment:
We only apply the clip if the content overflows the border.

Reply:
I think we need to add this as a comment here.